lib/constants/constants.go

```diff
@@ -774,6 +778,12 @@ var (
 	HubImageName = "hub"
 )
 
+// ExternalDNS formats the provided hostname as a wildcard A record for use
+// with external DNS provisioners.
+func ExternalDNS(hostname string) string {
```

Comment:
nit: I thought this would somehow use `ExternalDNSHostnameAnnotation` - otherwise it belongs in `lib/utils/net` or something like that.